glitter/blockadmin/blocks.py

```diff
@@ -105,6 +105,9 @@ def inner(request, *args, **kwargs):
         inner = csrf_protect(inner)
         return update_wrapper(inner, view)
 
+    def get_app_list(self, request):
+        return dict()
```

Comment:
So that we're all on the same page, for consistency - let's use `{}` for all dicts.